src/css/main.css

```diff
@@ -91,6 +94,7 @@ html, body {
     display: flex;
     justify-content: space-between; 
     align-content: center;
+    background: dodgerBlue;
```

Comment:
You're going from using color name alias, to `rgb`  and also `hex`  values (#ffff), we need to be consistent here and try to use format only. Maybe `rgb`  might be a better choice as we can add opacity in a simpler way?